manifests/config.pp

```diff
@@ -10,6 +10,19 @@
 
   $_ip = join($ferm::ip_versions, ' ')
 
+  if $facts['systemd'] { #fact provided by systemd module
+    if $ferm::install_method == 'vcsrepo' and $ferm::manage_service {
+      include systemd::systemctl::daemon_reload
```

Comment:
this isn't required. systemd::dropin_file does the notify:
https://github.com/camptocamp/puppet-systemd/blob/master/manifests/dropin_file.pp#L93-L103